kotsadm/pkg/updatechecker/updatechecker.go

```diff
@@ -305,3 +289,60 @@ func CheckForUpdates(appID string, deploy bool) (int64, error) {
 
 	return availableUpdates, nil
 }
+
+func GetReportingInfo(appID string) (*upstreamtypes.ReportingInfo, error) {
+	r := upstreamtypes.ReportingInfo{
+		AppID: appID,
+	}
+
+	downstreams, err := store.GetStore().ListDownstreamsForApp(appID)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to list downstreams for app")
+	}
+	r.ClusterID = downstreams[0].ClusterID
```

Comment:
this can be an empty array

Reply:
at this point? hmmm, i thought about that, well, doesn't hurt to handle that.